src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -20,13 +21,28 @@
 import seedu.address.model.Model;
 import seedu.address.model.ReadOnlyTaskWise;
 import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.task.Deadline;
 import seedu.address.model.task.Description;
+import seedu.address.model.task.Note;
+import seedu.address.model.task.Priority;
 import seedu.address.model.task.Status;
 import seedu.address.model.task.Task;
 import seedu.address.model.task.UniqueTaskList;
 
 
 public class SortCommandTest {
+    private static final Description FIRST_DESCRIPTION = new Description("oh no");
+    private static final Description SECOND_DESCRIPTION = new Description("an apple");
+    private static final Description THIRD_DESCRIPTION = new Description("xylophone");
+    private static final Status DEFAULT_STATUS = new Status();
+    private static final Status COMPLETE_STATUS = new Status(true);
+    private static final Status INCOMPLETE_STATUS = new Status(false);
+    private static final Note DEFAULT_NOTE = new Note("");
+    private static final Deadline FIRST_DEADLINE = new Deadline(LocalDateTime.of(2023, 1, 1, 18, 0));
+    private static final Deadline SECOND_DEADLINE = new Deadline(LocalDateTime.of(2023, 1, 1, 22, 0));
```

Comment:
With the new changes made to `Deadline`, should this be changed to:

```suggestion
    private static final Deadline FIRST_DEADLINE = Deadline.of(LocalDateTime.of(2023, 1, 1, 18, 0));
    private static final Deadline SECOND_DEADLINE = Deadline.of(LocalDateTime.of(2023, 1, 1, 22, 0));
```

Reply:
I have changed it accordingly! Thanks